jni/src/test/java/org/infinispan/client/jni/hotrod/JniTest.java

```diff
@@ -98,8 +98,7 @@ public static void main(String[] args) {
             "GetAllLocalTest.testBulkGetAfterLifespanExpire",
             "GetAllReplTest.testBulkGetAfterLifespanExpire",
             "RemoteCacheAdminTest.cacheReindexTest",                             // See HRCPP-469
-            "RemoteCacheAdminTest.cacheCreateWithEmbeddedConfigurationTest",     // conf to XML doesn't work here
-            "RemoteCacheAdminTest.testGetCacheNames"                             // HRCPP-475
```

Comment:
This test can't be used? Are there any other tests for this?

Reply:
This test needs the configuration to XML feature which is not implemented in C++ or C#